roles/ceph-monitor/tasks/monitoring.yml

```diff
@@ -19,6 +19,7 @@
 - name: ceph usage metrics
   sensu_metrics_check:
     name: ceph-usage-metrics
-    plugin: metrics-ceph.py
+    plugin: metrics-ceph-pool-capacity.rb
+    args='--scheme {{ monitoring.graphite.host_prefix }}'
```

Comment:
does this need to be `args='--scheme {{ monitoring.graphite.host_prefix }}.ceph' ?   and is it a per host or per cluster metric ?